src/javascript/app/pages/user/metatrader/metatrader.config.js

```diff
@@ -414,7 +415,6 @@ const MetaTraderConfig = (() => {
 
     const validations = () => ({
         new_account: [
-            { selector: fields.new_account.txt_name.id,          validations: [['req', { hide_asterisk: true }], 'letter_symbol', ['length', { min: 2, max: 101 }]] },
```

Comment:
Wait! I thought it was mandatory in the API to pass in `name`...

Reply:
this is for the validation, not the API

Reply:
since it's not an input field and is now a hardcoded value, we don't need any validation

Reply:
Ah perfect.